### PR TITLE
Add clause for non-information changes in metadata

### PR DESCRIPTION
**Pull request by matentzn**
In this time where the website is undergoing serious changes, it is not feasible to wait for metadata owners for review without slowing us down massively. This pull request amends our SOP for metadata changes with a special clause that allows an OBO Operations member to accept a pull request that does not change the information content of the metadata, i.e. merely changes the _presentation_ of the information. This is really important for us to do our job as website developers properly.

### CODE REVIEW

File: docs/SOP.md [hunk 1]
```diff
@@ -30,11 +30,12 @@ In general, the metadata record of an ontology in the OBO Foundry metadata regis
...
 2. This pull request is reviewed by a member of the OBO Foundry operations committee and may be rejected.
-3. The OBO Foundry operations committee member MUST then tag the listed contact person using their GitHub handle on the pull request requesting a review (initial request for feedback).
-4. After two weeks without activity, an OBO Foundry operations committee member will update the pull request and announce a prospective merge date no earlier than two weeks in the future.
-5. After at least 4 weeks have passed since the initial request for review:
+3. If the proposed change affects merely the _presentation_ of the web page (information moved from one section of the website to another; changed URLs from http to https, change in the formatting (dimensions) of the ontology logo, typos, the OBO Foundry operations committee member may choose to accept the pull request without waiting for further review (Note, this rule is important to not block website development for months at a time). If the previous case exists, the following points may be ignored.
```
Comment (matentzn) [inline docs/SOP.md:33]: This line is the only line that changed, @lschriml, @nataled, the other ones only appear changed due to the change in numbering.

**Comment by cthoyt** [comment]
I like this!

**Review by nataled — APPROVED** [review]

File: docs/SOP.md [hunk 2]
```diff
@@ -30,11 +30,12 @@ In general, the metadata record of an ontology in the OBO Foundry metadata regis
...
 2. This pull request is reviewed by a member of the OBO Foundry operations committee and may be rejected.
-3. The OBO Foundry operations committee member MUST then tag the listed contact person using their GitHub handle on the pull request requesting a review (initial request for feedback).
-4. After two weeks without activity, an OBO Foundry operations committee member will update the pull request and announce a prospective merge date no earlier than two weeks in the future.
-5. After at least 4 weeks have passed since the initial request for review:
+3. To prevent undue delays to website development, if the proposed change affects merely the presentation of the web page (for example, information is moved from one section of the website to another, URLs are changed from http to https, there is a change in the display/formatting of a page, typos) the OBO Foundry Operations Committee member may choose to accept the pull request without waiting for further review. In such case, the following steps may be ignored.
```
[the remark this answers is not in the document]
Comment (lschriml) [reply docs/SOP.md:33]: Agreed, this is a good step forward. As long as we do not loose any information.

**Review by lschriml — COMMENTED** [review]
Approved.

File: docs/SOP.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (matentzn) [reply docs/SOP.md:33]: Excellent, thank you!

**Pull request merged**